Add files via upload (#94)
PQCA/TAC: meeting-minutes/2025/2025-07-30-TAC.md

## **Voting Representative Attendance (*Alphabetical by 1st name*)**

### **Premier and General Member Representatives**

Brian Jarvis, Amazon Web Services Inc. \[TAC Chair\]  
Nicklas Körtge (IBM Research Zurich)   
Norman Ashley, Cisco 

* Yarkin Doroz, NVIDIA  
* Michael Maximilien (IBM) (Not attended)  
* JP Lomas (QRL) (Not attended)

### **Project Representatives**

* Matthias Kannwischer (TSR)

## **Non-Voting Representative (Member companies)**

* Joe Livingston (IBM)  
* Andreas Schade (IBM)   
* Prevek Sharma (University of waterloo) (Not attended)

### **LF Staff**

*  Hart Montgomery   
*  Balaji Ethirajulu (Not Attended)

### **Other Attendees**

> * Aditya Koranga (CORAN Labs) (TAC Vice chair)  

Reviewer: ```suggestion
* Aditya Koranga (NgKore) (TAC Vice chair)  
```

* Blog publication updates? (Tooling, PQCP both ready to publish)  
* Follow ups from 2025-07-16  
  \- Project lifecycle stage for OQS  
  \- OQS survey link  
* Lifecycle document annual review – Volunteer needed to shepherd the annual review  
* Proposed PQCA website updates  
  \- [https://github.com/PQCA/TAC/issues/90](https://github.com/PQCA/TAC/issues/90) \- Update to latest TAC roster  
  \- [https://github.com/PQCA/TAC/issues/91](https://github.com/PQCA/TAC/issues/91) \- Discoverability of PQCA wiki (link, subdomain, etc)  
* Project status updates (slides: link TBD)  
  \- OQS  
  \- PQCP  
  \- Tooling

## 

## **CBOM kit tool discussions** 

Before the official meeting began, **Sandip Dholakia** and **Andreas Schade** had a brief discussion regarding the **CBOM Kit Tool**. They plan to continue these discussions separately.

There was also a conversation about the potential of making the CBOM Kit a formal project. **Nicklas Kortge** and **Andreas Schade** raised questions, and **Hart Montgomery** provided insights into the process. Montgomery proposed setting up a dedicated meeting to discuss this further.

---

## **PQCA \- Blogposts**

> Two blog posts—Mlkem Native and CBOM Kit—were reviewed by both the TAC and the Content Review Committee. It was agreed that both would be published on the PQCA website and shared via the LinkedIn page.

Reviewer: ```suggestion
Two blog posts— mlkem-native and CBOMKit—were reviewed by both the TAC and the Content Review Committee. It was agreed that both would be published on the PQCA website and shared via the LinkedIn page.
```

---

## **Other Agenda topics**  

* **Project Lifecycle – OQS**:  
  **Jarvis** mentioned the project lifecycle stage for OQS. **Ashley** noted that this topic came up during the **TSC** discussions, but no decision has been finalized yet. Jarvis asked whether the TSC might address this in their next meeting; Norman responded that it's a possibility.

* **Annual Review of Project Lifecycle Document**:  
   Jarvis requested a volunteer to lead the review of the project lifecycle document. **Ashley** volunteered and inquired about the timeframe. Jarvis suggested aiming for the end **of August**. Montgomery offered support and encouraged Ashley to reach out with any questions.

* **PQCA Website and Wiki Update**:  
   Jarvis brought up updates to the PQCA website and its linked wiki page. Montgomery recommended sending the request to **Balaji Ethirajulu** and himself for processing.

---

## **OQS Update \- Norman Ashley**

* Lib OQS version 0.14 was released earlier in the month.  
* OQS provider 0.10 to be released shortly.  
* TSC meeting is coming up in the middle of August.

---

## **PQCP Update \- Matthias Kannwischer**

> * Ml-kem our arm backend is merged into AWS LC two days ago and that is a great milestone.  If you need a faster ARM ml-kem, it is now in AWS LC  

Reviewer: ```suggestion
* mlkem-native arm backend is merged into AWS-LC two days ago and that is a great milestone.  If you need a faster ARM ml-kem, it is now in AWS-LC  
```

> * For ml-dsa native, we have completed the cvmc proofs.  

Reviewer: ```suggestion
* For mldsa-native, CBMC proofs have been completed. 
```

> * Working on constant time hardening and we will be looking at performance to have the matching state of the art for SLH-DSA. There has been some work on integrating this into lib OQS.  

Reviewer: ```suggestion
* Working on constant time hardening and will be looking at performance to have the matching state of the art for SLH-DSA. There has been some work on integrating this into liboqs.  
```

  ---

## **Tools WG Update \- Andreas Schade**

> * We have two new version of the CBOM kit. CBOM version 2.1.9 and that is CBOM Kit service, that is one with graphical web interface that was demonstrated two weeks ago,  

Reviewer: ```suggestion
* Two new version of the CBOM tools: CBOMkit version 2.1.9, that is CBOM Kit service- the one with graphical web interface that was demonstrated two weeks ago,  
```

> * CBOM 2.1.1 for CBOM Kit action and that is the github action for workflows.  

Reviewer: ```suggestion
* 2.1.1 for CBOM Kit action and that is the github action for workflows.  
```
